buildkite/resource_pipeline.go

```diff
@@ -183,6 +183,7 @@ func (p *pipelineResource) Create(ctx context.Context, req resource.CreateReques
 	if len(plan.Teams) > 0 {
```

Comment:
Don't forget as part of this PR to add the deprecation message to the teams block on pipeline